react-app/src/api/distributionAPI.ts

```diff
@@ -67,16 +73,51 @@ export const useDistributionAPI = (): IDistributionAPI => {
     );
 
     return {
-      denom: coinMinimalDenom,
+      denom: CoinDenom,
       amount: convertMinimalTokenToToken(rewardAmount),
     };
-  }, [coinMinimalDenom, wallet, query]);
+  }, [wallet, query]);
+
+  const getTotalCommission = useCallback(async () => {
+    if (wallet.status !== ConnectionStatus.Connected) {
+      throw new Error("Wallet not connected");
+    }
+
+    const address = translateAddress(wallet.account.address, "likevaloper");
```

Comment:
we are going to opensource it and it's possible for others to use prefix other than `likevaloper`, could be `xvaloper` and `yvaloper`, let's put this into config